docs/ISSUE_TEMPLATE.md

```diff
@@ -6,9 +6,10 @@
 ## Why
 > Explain why you think this should be added to the GOV.UK Design System.
 >
-> - Is it needed by multiple services across government?
-> - Does it meets the needs of the users of those services?
-> - Could those needs be better met by something already in the Design System?
+> - What evidence do you have that it's needed by multiple services across government?
```

Comment:
Is there any other evidence than 'it is used by X and Y'?

In which case, should this be 'Which services use this pattern? (At least 2)'

Reply:
I think we should leave it open for other kinds of evidence. For example we've surveyed designers before to ask what patterns they need.

Reply:
Agreed. I spoke to Joe about this last night and he's happy if we leave it open - to begin with at least.